pyscript/shape2raster.py

```diff
@@ -0,0 +1,105 @@
+import geopandas as gpd
+import numpy
+import os
+from osgeo import gdal
+from scipy import interpolate
+import pykrige as pykg
+
+
+#  class for writing raster data
+class Rasterfun(object):
```

Comment:
No need to inherit from object, this is fine (besides the class name)
```suggestion
class Rasterfun:
```